### PR TITLE
@alloy => warning for short passwords, invalid emails on create account 📱

### LINKED ISSUE #238: [Signup] No feedback when a password doesn't validate.
E.g. with a password such as `test` (I assume too short) tapping submit doesn’t provide any visual feedback about it performing any work and/or letting the user know that the password doesn’t validate.

### PR DESCRIPTION
**Pull request by jorystiefel**
Adds a warning banner if either the password is too short (<6 chars) or the email fails a very simple validation check (no "@").

I've modified the `ARWarningView` class so that the warning icon is tinted to match the view's textColor, this way we can change the banner color and the icon and text will always match.

I've also replaced the AttentionIcon resource with a version that is opaque, should we need semi-transparency (we don't currently), we can set it in code.

Closes #238.

### CODE REVIEW

File: Artsy/View_Controllers/Login_and_Onboarding/Onboarding_stages/3_-_Sign_Up_+_Log_In/ARCreateAccountViewController.m [hunk 1]
```diff
@@ -259,6 +262,45 @@ - (void)accountExists:(NSString *)source
     [alert show];
 }
 
+- (void)showWarning:(NSString *)msg
```
Comment (orta) [inline Artsy/View_Controllers/Login_and_Onboarding/Onboarding_stages/3_-_Sign_Up_+_Log_In/ARCreateAccountViewController.m]: this should take an `animated:(BOOL)animates` - which means you can then change: `[UIView animateWithDuration` to `[UIView animateIf...`

File: Artsy/View_Controllers/Login_and_Onboarding/Onboarding_stages/3_-_Sign_Up_+_Log_In/ARCreateAccountViewController.m [hunk 2]
```diff
@@ -259,6 +262,45 @@ - (void)accountExists:(NSString *)source
...
+                           predicate:nil];
+
+    [UIView animateWithDuration:0.15 animations:^{
+        self.warningView.alpha = 1;
+    }];
+
+    [self performSelector:@selector(removeWarning) withObject:nil afterDelay:5];
+}
+
+- (void)removeWarning
```
Comment (orta) [inline Artsy/View_Controllers/Login_and_Onboarding/Onboarding_stages/3_-_Sign_Up_+_Log_In/ARCreateAccountViewController.m]: same as above

**Comment by alloy** [comment]
What @orta said, otherwise this looks good, I think that test failure is a known flaky one.

**Comment by orta** [comment]
Looks good, let me know when you get in tomorrow and I'll jam with you on some tests :+1:

**Comment by jorystiefel** [comment]
I think this is good to go @alloy

**Comment by alloy** [comment]
Looks great :clap: 

Have you tested this extensively and on a device? If so, add the :iphone: emoji to the title.

**Comment by jorystiefel** [comment]
All set @alloy

**Comment by alloy** [comment]
Perfect, thanks!

**Pull request merged**